### DISCUSSION TITLE
Is there any way to use colSpan in Header

### DISCUSSION THREAD

**Discussion by quyle92** [KevinVandy/material-react-table#662] [Q&A]
Is there a way to represent using colSpan in Header as shown below?

![image](https://github.com/KevinVandy/material-react-table/assets/59790219/8303fd16-6b3c-458d-95a9-372f40a9a935)

I found something similar: https://github.com/KevinVandy/material-react-table/discussions/269
I guess props like `muiTableHeadCellProps` or `muiTableHeadRowProps` might be helpful but have no idea how to implement.

Can you please give me a hint?

**Comment by mikemayster** [comment]
Hello,

How did you do the header columns under Number of Achivement?

Can you give me an example pls

**Reply by quyle92** [reply]
I used Ag Grid col-span, which is a bit harder but it worked the way I wanted.
Here is the link: [https://www.ag-grid.com/react-data-grid/column-spanning/](https://www.ag-grid.com/react-data-grid/column-spanning/)

**Reply by mikemayster** [reply]
Thank you, but I would like to use something from this component.

**Comment by pfmartins-zartis** [comment]
Having same need here, anything new?
